commit d63230d73e710b6ebc8b77fc99f9571687852f12
Merge: 548b450 8e8b439
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Wed Nov 16 00:16:36 2016 -0600

    Merge pull request #5192 from Kaibob2/patch-7
    
    Printer Stats / Completed prints line doesn't fit in display (when german)

diff --git a/Marlin/ultralcd.cpp b/Marlin/ultralcd.cpp
index 1642639..2216a9f 100755
--- a/Marlin/ultralcd.cpp
+++ b/Marlin/ultralcd.cpp
@@ -2011,7 +2011,7 @@ void kill_screen(const char* lcd_msg) {
 
         START_SCREEN();                                                                                // 12345678901234567890
         STATIC_ITEM(MSG_INFO_PRINT_COUNT ": ", false, false, itostr3left(stats.totalPrints));          // Print Count: 999
-        STATIC_ITEM(MSG_INFO_COMPLETED_PRINTS"  : ", false, false, itostr3left(stats.finishedPrints)); // Completed  : 666
+        STATIC_ITEM(MSG_INFO_COMPLETED_PRINTS": ", false, false, itostr3left(stats.finishedPrints)); // Completed  : 666
 
         duration_t elapsed = stats.printTime;
         elapsed.toString(buffer);
